lib/sum_of_depository_balances_latest/main.py

```diff
@@ -62,7 +62,7 @@ async def get_sum_of_depository_balances_latest(
             balances_flattened.append(balance_dict)
     
     # STEP 4: Here we sort by timestamp so latest balances are on top
-    balances_flattened = sorted(balances_flattened, key=lambda x: x["ts"], reverse=True)
+    balances_flattened = sorted(balances_flattened, key=lambda x: x["timestamp"], reverse=True)
```

Comment:
This is sorting lexicographically since `timestamp` is not `datetime` aware, right?

Any cases where that could lead to unexpected behavior?

Reply:
Yes, it's a lexicographic sort. Given how we are formatting the string (timestamp), I think this implementation will behave as expected. Tested locally with a handful of "edge" cases, to be double sure

Reply:
Only place I could see it failing is timezone differences but we return everything in UTC (`+00:00`) so should be fine.